src/lazy/any_encoding.rs

```diff
@@ -550,6 +550,10 @@ mod tests {
                 reader.next()?.expect_value()?.read()?,
                 RawValueRef::Int(5.into())
             );
+            assert_eq!(
+                reader.next()?.expect_value()?.read()?,
+                RawValueRef::Timestamp(Timestamp::with_year(2023).with_month(8).build()?)
+            );
```

Comment:
🗺️ This simple test confirms that the `LazyRawAnyReader` can surface timestamps as expected. More rigorous correctness testing happens later in the diff.